docs/private-app-create.md

```diff
@@ -0,0 +1,92 @@
+## Glossary
```

Comment:
Is there a reason to duplicate the Glossary on both the welcome/what page and the create page?

Reply:
That was an error. I hae moved the glossary out of the create page and put it in the general "What is a private app" page :)